### ISSUE TITLE
Learning rate scaling rule for different batch sizes

### ISSUE THREAD

**Issue by qizhuli** [SwinTransformer/Swin-Transformer-Semantic-Segmentation#34]
Hi there! I was trying to train your model on 4 cards, by increasing the number of samples per card to 4. The total batch size = 4x4 = 8x2 = 16, which is the same as your original setting. Everything else is kept the same. Do you think that would be equivalent to your 8-card experiment?

More generally, when the total batch size is changed, do you expect the linear LR scaling rule presented in [this report](https://arxiv.org/pdf/1706.02677.pdf) to hold true for Swin experiments, which use AdamW instead of SGD? If not, can you recommend a way for it? Many thanks in advance!
